cmake_modules/commonSetup.cmake

```diff
@@ -77,7 +77,13 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
   else()
     option(USE_NUMA "Configure use of numa" ON)
   endif()
-  option(USE_NATIVE_LIBRARIES "Search standard OS locations for thirdparty libraries" ON)
+
+  IF (WIN32)
+     option(USE_NATIVE_LIBRARIES "thirdparty libraries are in EXTERNALS_DIRECTORY" OFF)
```

Comment:
The text of this option says the opposite of what the option does, I think. Seems to me the option's text should remain the same for both platforms, it's just the default value you are wanting to change.

Reply:
The message tells where the 3rd party library is when the USE_NATIVE_LIBRARIES is set to default (which is OFF on Windows, and ON on other platform). I agree it is confusion, and will change it to be the same.